src/utils/functions/getDonationLateStatus.test.ts

```diff
@@ -0,0 +1,54 @@
+import { DateTime } from 'luxon';
+import { ContactLateStatusEnum } from '../../components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/ContactLateStatusLabel';
+import { getDonationLateStatus } from './getDonationLateStatus';
+
+const mockNow = DateTime.fromISO('2025-06-15T12:00:00Z');
+jest.spyOn(DateTime, 'now').mockReturnValue(mockNow as DateTime<true>);
```

Comment:
By default, "now" is mocked to January 1, 2020 (it's set up in `__tests__/util/setup.ts`). Can you either update your tests to be relative to that "now" or do the mocking like this:

```ts
Settings.now = () => new Date(2025, 5, 15).valueOf();
```